test/files/neg/t2102.check

```diff
@@ -1,6 +1,6 @@
 t2102.scala:2: error: type mismatch;
- found   : java.util.Iterator[Int]
- required: scala.collection.Iterator[_]
+ found   : Iterator[Int] (in java.util)
+ required: Iterator[_] (in scala.collection)
```

Comment:
Based on this idea, I think a small tweak to the current format could be as effective, and more compact:
```
found   :        java.util.Iterator[Int]
required: scala.collection.Iterator[_]
```